client/core/trade.go

```diff
@@ -1054,6 +1056,13 @@ func (t *trackedTrade) revokeMatch(matchID order.MatchID, fromServer bool) error
 
 	// Notify the user of the failed match.
 	corder := t.coreOrderInternal() // no cancel order
+	// For the frontend, call this a complete match so it doesn't say "settling".
+	for _, match := range corder.Matches {
+		if bytes.Equal(match.MatchID, matchID[:]) {
+			match.Status = order.MatchComplete
+			break
+		}
+	}
```

Comment:
We don't want to set it as complete in the DB data structures to make this persistent, and any subsequent order status update will not reflect this, as such it is a transient workaround to the lack of revoked match status or 
What's the best solution to prevent the browser from showing this order as "settling" (no fault user) or "revoked/settling" (at fault user) on account of the match being viewed as active?  Perhaps a `core.Match.Revoked` flag that is checked by `hasLiveMatches` in orderutil.js?

Reply:
`core.Match.Revoked` and mods to `hasLiveMatches` seems to have worked out. https://github.com/decred/dcrdex/pull/725/commits/0a97501fe1e41b2fdbd6112501ff955911ca91c3
This is mostly unrelated to this PR though, as this pertains to revoked matches, not revoked orders/